custom/icds_reports/tests/agg_tests/reports/test_bihar_api_demographics.py

```diff
@@ -19,27 +20,21 @@ def test_file_content(self):
         )
         first_person_case = data[0]
         self.assertCountEqual(
-            {"state_name": "Bihar", "state_site_code": "st1",
-             "district_name": "d1", "district_site_code": "d1",
-             "block_name": "b2", "block_site_code": "b2",
-             "supervisor_name": "s4", "supervisor_site_code": "s4",
-             "awc_name": "a36", "awc_site_code": "a36",
-             "ward_number": "", "household_id": '0c05ae24-3d23-4382-a346-8d053c1ec81c',
-             "household_name": None, "hh_reg_date": None,
-             "hh_num": None, "hh_gps_location": None,
-             "hh_caste": None, "hh_bpl_apl": None,
-             "hh_minority": None, "hh_religion": None,
-             "hh_member_number": None,
-             "person_id": "008b146c-0f21-4506-8800-6cdf5f0d04fc",
-             "person_name": None, "has_adhaar": 0,
-             "bank_account_number": '123456789', "ifsc_code": 'testcode1234',
-             "age_at_reg": 28, "dob": "1988-04-07", "gender": "F",
-             "blood_group": 'APos', "disabled": None,
-             "disability_type": None, "referral_status": 'reffered',
-             "migration_status": None, "resident": 1,
-             "registered_status": None, "rch_id": 1234,
-             "mcts_id": 213, "phone_number": None,
-             "date_death": None, "site_death": None,
-             "closed_on": None, "reason_closure": None},
+            {'state_name': 'st1', 'state_site_code': 'st1', 'district_name': 'd1', 'district_site_code': 'd1',
```

Comment:
how did this changed totally instead of just some field changing?. it looks this is different row than it was before.

Reply:
Formatting is differnet.But the data is same

Reply:
fixed the formatting